wazo_call_logd/database/queries/call_log.py

```diff
@@ -114,40 +116,40 @@ def count_in_period(self, params):
 
         return {'total': total, 'filtered': filtered}
 
-    def _apply_user_filter(self, query, params):
-        if params.get('me_user_uuid'):
-            me_user_uuid = params['me_user_uuid']
+    def _apply_user_filter(self, query: Query, params: dict[str, Any]) -> Query:
+        if me_user_uuid := params.get('me_user_uuid'):
             query = query.filter(
-                CallLog.participant_user_uuids.contains(str(me_user_uuid))
+                CallLog.participants.any(
+                    CallLogParticipant.user_uuid == str(me_user_uuid)
+                )
             )
         return query
 
-    def _apply_filters(self, query, params):
-        if params.get('start'):
-            query = query.filter(CallLog.date >= params['start'])
-        if params.get('end'):
-            query = query.filter(CallLog.date < params['end'])
+    def _apply_filters(self, query: Query, params: dict[str, Any]) -> Query:
+        if start_date := params.get('start'):
+            query = query.filter(CallLog.date >= start_date)
+        if end_date := params.get('end'):
+            query = query.filter(CallLog.date < end_date)
 
-        if params.get('call_direction'):
-            query = query.filter(CallLog.direction == params['call_direction'])
+        if call_direction := params.get('call_direction'):
+            query = query.filter(CallLog.direction == call_direction)
 
-        if params.get('cdr_ids'):
-            query = query.filter(CallLog.id.in_(params['cdr_ids']))
+        if cdr_ids := params.get('cdr_ids'):
+            query = query.filter(CallLog.id.in_(cdr_ids))
 
-        if params.get('id'):
-            query = query.filter(CallLog.id == params['id'])
+        if call_log_id := params.get('id'):
+            query = query.filter(CallLog.id == call_log_id)
 
-        if params.get('search'):
+        if search := params.get('search'):
             filters = (
-                sql.cast(column, sa.String).ilike('%%%s%%' % params['search'])
+                sql.cast(column, sa.String).ilike(f'%{search}%')
```

Comment:
Noice, I will have to start thinking with walruses too

Reply:
Currently only safe in the Bullseye branch, but yes. :)

Reply:
> > For future PRs, splitting commits logically(e.g. formatting separately from code changes) can be much help for reviewers(e.g. I would have looked at commits excluding formatting changes).
> 
> The formatting changes were only committed because excluding them would have caused a commit that was known to fail the CI. Since black has changed. Anyways, 4-5 removed empty lines aren't going to cause a huge impact to readability. Also this PR is still in progress.

Oups, ignore my early approval then